heartbeat/monitors/wrappers/summarizer/plugstatestat.go

```diff
@@ -56,20 +56,24 @@ func (ssp *BrowserStateStatusPlugin) EachEvent(event *beat.Event, jobErr error)
 }
 
 func (ssp *BrowserStateStatusPlugin) BeforeSummary(event *beat.Event) BeforeSummaryActions {
+	ssp.cssp.js.Status = monitorstate.StatusDown
```

Comment:
If we already set the Status here, it can be safely removed from the common BeforeSummary - https://github.com/elastic/beats/blob/a51b909b2ce4e504ef6e318c4ea4698613894cce/heartbeat/monitors/wrappers/summarizer/plugstatestat.go#L144-L148